_includes/examples/carriers/ups/ship_webservice/misc/access_point_delivery.html

```diff
@@ -46,50 +48,38 @@ <h4 id="ship-webservice---access-point-delivery" class="text-blue">
 {% highlight json %}
 {
   "from": {
-    "first_name": "Serge",
-    "last_name": "Sender",
-    "company": "Sender Corp.",
-    "street": "Sender Str.",
-    "street_no": "99",
-    "zip_code": "20148",
-    "city": "Hamburg",
-    "country": "DE"
+      "first_name": "Serge",
+      "last_name": "Sender",
+      "company": "Sender Corp.",
+      "street": "Sender Str.",
+      "street_no": "99",
+      "zip_code": "20148",
+      "city": "Hamburg",
+      "country": "DE"
```

Comment:
All JSON lines are now indented with two tabs or 4 spaces. Is that necessary?

Reply:
Fixed in [76d9947](https://github.com/shipcloud/shipcloud-developer-portal/pull/296/commits/76d9947974f94c077cd13758207d89839618596d)